packages/app/src/cli/services/context.ts

```diff
@@ -62,8 +62,8 @@ export const appFromId = async (options: AppFromIdOptions): Promise<Organization
       organizationId = org.id
     }
   }
-  const app = await developerPlatformClient.appFromId({
-    id: options.id ?? 'no-id-available',
+  const app = await developerPlatformClient.appFromIdentifiers({
+    id: options.id,
```

Comment:
should this still be supported? I thought we were _only_ identifying by API key now?